README.md

```diff
@@ -516,6 +586,8 @@ The first time you create a container out of the xwiki image, a shell script (`/
 -	`DB_HOST`: The name of the host (or docker container) containing the database. Default is "db".
 -	`INDEX_HOST`: The hostname of an externally configured Solr instance. Defaults to "localhost", and configures an embedded Solr instance.
 -	`INDEX_PORT`: The port used by an externally configured Solr instance. Defaults to 8983.
+- `CLUSTER`: Set it to "true" to enable clustering.
+- `CLUSTER_CHANNEL`: The JGroups channel name.
```

Comment:
Missing some tabs I think. BTW make sure to use tabs as this is the format for dockerhub (even though we stopped publishing the doc there, it would be better to continue using the format used in the readme already).

Reply:
okay thanks, done